docs/ReleaseNotes.html

```diff
@@ -115,9 +115,14 @@ <h1 align="center">Password Safe 3.41.1 Release Notes ???????? 201?</h1>
 <h4><a name="V3.41.1Changes"></a>Changes to Existing Features in V3.41.1</h4>
 <table>
 <tbody>
-<tr>
-<td align="center" width="100"><a href="nojs.html" onclick="FR(816);return false;">816</a></td>
-<td>Set installed version in uninstall section of Registry for Windows version.</td>
+<tr>
+<td align="center" width="100"><a href="nojs.html" onclick="FR(819);return false;">819</a></td>
+<td>When an entry is duplicated directly or by being in a group being duplicated, its

```

Comment:
Phrasing is confusing, as "its" is ambiguous. Suggest something like "a duplicated entry's creation time is set to the time of duplication. All other times are copied from the original."